server/cla.go

```diff
@@ -54,80 +59,108 @@ func (s *Server) checkCLA(pr *model.PullRequest) {
 		return
 	}
 
+	body, errCSV := s.getCSV()
```

Comment:
Out of scope of this PR: Cache the csv for one minute

Reply:
:100: